curvefs/src/client/common/common.cpp

```diff
@@ -99,6 +101,44 @@ WarmupType GetWarmupType(const std::string& type) {
     return ret;
 }
 
+using ::curve::common::StringToUll;
+
+// if direction is true means '+', false means '-'
+bool AddUllStringToFirst(std::string *first, uint64_t second, bool direction) {
```

Comment:
I think the function is confuse, it mixs much actions, maybe we can split it into more single functions.

Reply:
the code is moved from curvefs/src/client/xattr_manager.cpp, I think it's bettle not to modify the logic of other people's code.